browser/ui/views/frame/split_view/brave_multi_contents_view_separator_delegate.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_FRAME_SPLIT_VIEW_BRAVE_MULTI_CONTENTS_VIEW_SEPARATOR_DELEGATE_H_
+#define BRAVE_BROWSER_UI_VIEWS_FRAME_SPLIT_VIEW_BRAVE_MULTI_CONTENTS_VIEW_SEPARATOR_DELEGATE_H_
+
+// Use different delegate for BraveMultiContentsView.
+// Although we have SplitViewSeparatorDelegate that inherits
+// views::ResizeAreaDelegate but BraveMultiContentsView can't use it because it
+// already inherits views::ResizeAreaDelegate.
+class BraveMultiContentsViewSeparatorDelegate {
```

Comment:
ah that sucks :/ 

I don't know that implementing two different version of whats pretty much the same interface is the way we want to go here.

What if instead, we have `SplitViewSeparator` take a `views::ResizeAreaDelegate` as well as a `SplitViewSeparatorDelegate` (which would only have `OnSeparatorDoubleClicked`).

That way we wouldn't need to have two separator code paths for the separate delegates - each one would just be for one part.

Reply:
Yeah, separating current interface looks better :) Done.